foo_uie_console/main.cpp

```diff
@@ -267,6 +277,11 @@ void ConsoleWindow::set_config(stream_reader* reader, t_size p_size, abort_callb
         int32_t edge_style{};
         reader->read_lendian_t(edge_style, abort);
         m_edge_style = EdgeStyle{edge_style};
+
```

Comment:
I still feel this is very much a compromise. I didn't want to meddle with `current_config_version`, but handling a single value like that looks out of place.

Reply:
Having a more elaborate data structure in the first place would've helped, but at the same time it doesn't seem worth it for just two values...

Feel free to add a comment explaining the reason for it though (most probably the new version will be 2.0.0 if you want to refer to it in the comment).

Reply:
Well, I don't know the API's well enough to be proposing a more sophisticated and universal mechanism. And I agree it wouldn't be worth it for this single case alone. I bet this will get refactor by the time 2.0.0 comes along.
A comment would suffice for now.